panda/src/putil/loaderOptions.h

```diff
@@ -48,16 +48,23 @@ class EXPCL_PANDA_PUTIL LoaderOptions {
     TF_allow_compression = 0x0200,  // Consider compressing RAM image
     TF_no_filters        = 0x0400,  // disallow using texture pool filters
   };
-
+  
   LoaderOptions(int flags = LF_search | LF_report_errors);
   constexpr LoaderOptions(int flags, int texture_flags);
 
   INLINE void set_flags(int flags);
   INLINE int get_flags() const;
   MAKE_PROPERTY(flags, get_flags, set_flags);
-
   INLINE void set_texture_flags(int flags);
   INLINE int get_texture_flags() const;
+  INLINE void set_texture_format(int format);
+  INLINE int get_texture_format() const;
+  INLINE void set_texture_compress(int  compress);
+  INLINE int get_texture_compress() const;
```

Comment:
I'm not really a fan of abbreviating an existing property, so I would prefer if this were named `texture_compression`

Reply:
Fixed